src/test/java/uk/gov/hmcts/cmc/claimstore/events/paindinfull/PaidInFullCitizenNotificationHandlerTest.java

```diff
@@ -45,11 +48,12 @@ public void setUp() {
     public void sendNotificationsSendsNotificationsToDefendant() throws NotificationClientException {
 
         PaidInFullEvent event = new PaidInFullEvent(SampleClaim.getDefault());
+        Response fullDefenceResponse = SampleResponse.FullDefence.builder().build();
+        Claim claim = SampleClaim.getWithResponse(fullDefenceResponse);
```

Comment:
The above two lines are not required.

Reply:
Test fails without it as getWithResponse() is needed to initialise the defendant's email in the sample claim

Reply:
Please pass this claim object in event constructor.